src/Features/ERDDAP/Platform/Observations/CurrentConditions/data_card.tsx

```diff
@@ -118,26 +113,26 @@ export const DataCardDisplay: React.FunctionComponent<DataCardDisplayProps> = ({
 
   const data = readings.map((r) => ({
     ...r,
-    reading: round(data_converter.convertToNumber(r.reading, unit_system), 2),
+    reading: round(data_converter.convertToNumber(r.reading, unitSystem), 2),
```

Comment:
Codacy found an issue: [Identifier 'data_converter' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6227039)